### PR TITLE
go rewrite - script to convert custom templates

### PR DESCRIPTION
**Pull request by c2thorn**
```
make refresh-go VERSION=beta OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta" PRODUCT=compute
```

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/main.go [hunk 1]
```diff
@@ -28,28 +28,36 @@ var version = flag.String("version", "", "optional version name. If specified, t
...
 	if outputPath == nil || *outputPath == "" {
-		log.Fatalf("No output path specified")
+		log.Printf("No output path specified, exiting")
+		return
 	}
 
 	if version == nil || *version == "" {
-		log.Fatalf("No version specified")
+		log.Printf("No version specified, assuming ga")
+		*version = "ga"
```
Comment (zli82016) [inline mmv1/main.go:56]: Is beta better than ga to be assumed?

Comment (c2thorn) [reply mmv1/main.go:56]: ruby versions currently assumes ga  : https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/compiler.rb#L46

Comment (zli82016) [reply mmv1/main.go:56]: Oh, got it.

**Review by zli82016 — APPROVED** [review]
LGTM

**Pull request merged**